backend/src/main/java/de/bund/digitalservice/ris/norms/application/validator/SingleModValidator.java

```diff
@@ -0,0 +1,80 @@
+package de.bund.digitalservice.ris.norms.application.validator;
+
+import de.bund.digitalservice.ris.norms.domain.entity.CharacterRange;
+import de.bund.digitalservice.ris.norms.domain.entity.Mod;
+import de.bund.digitalservice.ris.norms.domain.entity.Norm;
+import de.bund.digitalservice.ris.norms.domain.entity.TextualMod;
+import de.bund.digitalservice.ris.norms.utils.exceptions.ValidationException;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+import org.w3c.dom.Node;
+
+/**
+ * A custom validator for validating single mods that are being updated on the amending law. This
+ * validator uses the ZF0 version of the target law targeted by the mod
+ */
+@Component
+public class SingleModValidator implements Validator {
+
+  public static final ValidatorName NAME = ValidatorName.SINGLE_MOD;
+
+  @Override
+  public void validate(final Object... args) throws ValidationException {
```

Comment:
Could you please add a doc-comment explaining what `args[0]` and `args[1]` is supposed to contain.